app/Http/Requests/BotUpdateRequest.php

```diff
@@ -25,22 +25,26 @@ public function authorize()
      */
     public function rules()
     {
-        return [
+        $rules = [
             'name' => [
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Requests/BotUpdateRequest.php</br>**Line**: 29</br>Each line in an array declaration must end in a comma